[2019-07-10T11:40:49.517] [ERROR] plan - Unknown column 'qualifiedDemands' in 'field list' { SequelizeDatabaseError: Unknown column 'qualifiedDemands' in 'field list'
    at Query.formatError (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/dialects/mysql/query.js:239:16)
    at Query.handler [as onResult] (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/dialects/mysql/query.js:46:23)
    at Query.execute (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/commands/command.js:30:14)
    at Connection.handlePacket (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:449:32)
    at PacketParser.Connection.packetParser.p [as onPacket] (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:72:12)
    at PacketParser.executeStart (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/packet_parser.js:75:16)
    at Socket.Connection.stream.on.data (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:79:25)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
    at TCP.onread (net.js:607:20)
From previous event:
    at Query.run (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/dialects/mysql/query.js:39:12)
    at runHooks.then (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/sequelize.js:635:29)
From previous event:
    at Promise.try.then.connection (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/sequelize.js:635:12)
From previous event:
    at Promise.resolve.retry (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/sequelize.js:632:10)
    at /Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/retry-as-promised/index.js:70:21
    at new Promise (<anonymous>)
    at retryAsPromised (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/retry-as-promised/index.js:60:10)
    at Promise.try (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/sequelize.js:619:30)
From previous event:
    at Sequelize.query (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/sequelize.js:576:23)
    at QueryInterface.select (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/query-interface.js:1120:27)
    at Promise.try.then.then.then (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/model.js:1749:34)
    at runCallback (timers.js:794:20)
    at tryOnImmediate (timers.js:752:5)
    at processImmediate [as _immediateCallback] (timers.js:729:5)
From previous event:
    at Function.findAll (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/model.js:1746:8)
    at queryData (/Users/zhanghaiquan/Documents/my/amorphous_node/controller/plan.js:30:36)
    at Layer.handle [as handle_request] (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/express/lib/router/layer.js:95:5)
    at /Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/express/lib/router/index.js:335:12)
    at next (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/express/lib/router/index.js:275:10)
    at Function.handle (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/express/lib/router/index.js:174:3)
    at router (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/express/lib/router/index.js:47:12)
    at Layer.handle [as handle_request] (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/express/lib/router/index.js:317:13)
    at /Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/express/lib/router/index.js:335:12)
    at next (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/express/lib/router/index.js:275:10)
    at checkLogin (/Users/zhanghaiquan/Documents/my/amorphous_node/middleware/checkLogin.js:9:5)
    at Layer.handle [as handle_request] (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/express/lib/router/index.js:317:13)
    at /Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/express/lib/router/index.js:335:12)
    at next (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/express/lib/router/index.js:275:10)
    at /Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/express-session/index.js:489:7
  name: 'SequelizeDatabaseError',
  parent: 
   { Error: Unknown column 'qualifiedDemands' in 'field list'
    at Packet.asError (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/packets/packet.js:684:17)
    at Query.execute (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/commands/command.js:28:26)
    at Connection.handlePacket (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:449:32)
    at PacketParser.Connection.packetParser.p [as onPacket] (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:72:12)
    at PacketParser.executeStart (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/packet_parser.js:75:16)
    at Socket.Connection.stream.on.data (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:79:25)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
    at TCP.onread (net.js:607:20)
     code: 'ER_BAD_FIELD_ERROR',
     errno: 1054,
     sqlState: '42S22',
     sqlMessage: 'Unknown column \'qualifiedDemands\' in \'field list\'',
     sql: 'SELECT `planId`, `date`, `approved`, `remark`, `fileNumber`, `castId`, `team`, `taskOrder`, `ribbonTypeName`, `ribbonWidth`, `client`, `furnace`, `alloyWeight`, `castTime`, `rawWeight`, `orderThickness`, `orderLaminationFactor`, `orderRibbonToughnessLevels`, `orderAppearenceLevels`, `qualifiedDemands`, `realRibbonWidth`, `createdAt`, `updatedAt` FROM `plan` AS `plan` WHERE `plan`.`date` = \'2019-07-10\' AND `plan`.`castId` = \'6\' ORDER BY `plan`.`furnace` ASC;' },
  original: 
   { Error: Unknown column 'qualifiedDemands' in 'field list'
    at Packet.asError (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/packets/packet.js:684:17)
    at Query.execute (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/commands/command.js:28:26)
    at Connection.handlePacket (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:449:32)
    at PacketParser.Connection.packetParser.p [as onPacket] (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:72:12)
    at PacketParser.executeStart (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/packet_parser.js:75:16)
    at Socket.Connection.stream.on.data (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:79:25)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
    at TCP.onread (net.js:607:20)
     code: 'ER_BAD_FIELD_ERROR',
     errno: 1054,
     sqlState: '42S22',
     sqlMessage: 'Unknown column \'qualifiedDemands\' in \'field list\'',
     sql: 'SELECT `planId`, `date`, `approved`, `remark`, `fileNumber`, `castId`, `team`, `taskOrder`, `ribbonTypeName`, `ribbonWidth`, `client`, `furnace`, `alloyWeight`, `castTime`, `rawWeight`, `orderThickness`, `orderLaminationFactor`, `orderRibbonToughnessLevels`, `orderAppearenceLevels`, `qualifiedDemands`, `realRibbonWidth`, `createdAt`, `updatedAt` FROM `plan` AS `plan` WHERE `plan`.`date` = \'2019-07-10\' AND `plan`.`castId` = \'6\' ORDER BY `plan`.`furnace` ASC;' },
  sql: 'SELECT `planId`, `date`, `approved`, `remark`, `fileNumber`, `castId`, `team`, `taskOrder`, `ribbonTypeName`, `ribbonWidth`, `client`, `furnace`, `alloyWeight`, `castTime`, `rawWeight`, `orderThickness`, `orderLaminationFactor`, `orderRibbonToughnessLevels`, `orderAppearenceLevels`, `qualifiedDemands`, `realRibbonWidth`, `createdAt`, `updatedAt` FROM `plan` AS `plan` WHERE `plan`.`date` = \'2019-07-10\' AND `plan`.`castId` = \'6\' ORDER BY `plan`.`furnace` ASC;' }
[2019-07-10T11:48:33.560] [ERROR] user - Validation error { SequelizeUniqueConstraintError: Validation error
    at Query.formatError (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/dialects/mysql/query.js:218:16)
    at Execute.handler [as onResult] (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/dialects/mysql/query.js:46:23)
    at Execute.execute (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/commands/command.js:30:14)
    at Connection.handlePacket (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:449:32)
    at PacketParser.Connection.packetParser.p [as onPacket] (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:72:12)
    at PacketParser.executeStart (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/packet_parser.js:75:16)
    at Socket.Connection.stream.on.data (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:79:25)
    at Socket.emit (events.js:189:13)
    at addChunk (_stream_readable.js:284:12)
    at readableAddChunk (_stream_readable.js:265:11)
    at Socket.Readable.push (_stream_readable.js:220:10)
    at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
From previous event:
    at Query.run (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/dialects/mysql/query.js:39:12)
    at runHooks.then (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/sequelize.js:635:29)
From previous event:
    at Promise.try.then.connection (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/sequelize.js:635:12)
From previous event:
    at Promise.resolve.retry (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/sequelize.js:632:10)
    at /Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/retry-as-promised/index.js:70:21
    at new Promise (<anonymous>)
    at retryAsPromised (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/retry-as-promised/index.js:60:10)
    at Promise.try (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/sequelize.js:619:30)
From previous event:
    at Sequelize.query (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/sequelize.js:576:23)
    at QueryInterface.insert (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/query-interface.js:885:27)
    at Promise.try.then.then.then (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/model.js:3812:52)
    at runCallback (timers.js:705:18)
    at tryOnImmediate (timers.js:676:5)
    at processImmediate (timers.js:658:5)
From previous event:
    at model.save (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/model.js:3789:8)
    at Function.create (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/model.js:2216:8)
    at createUser (/Users/zhanghaiquan/Documents/my/amorphous_node/controller/user.js:102:23)
  name: 'SequelizeUniqueConstraintError',
  errors:
   [ ValidationErrorItem {
       message: 'roleId_UNIQUE must be unique',
       type: 'unique violation',
       path: 'roleId_UNIQUE',
       value: '7',
       origin: 'DB',
       instance: [user],
       validatorKey: 'not_unique',
       validatorName: null,
       validatorArgs: [] } ],
  fields: { roleId_UNIQUE: '7' },
  parent:
   { Error: Duplicate entry '7' for key 'roleId_UNIQUE'
       at Packet.asError (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/packets/packet.js:684:17)
       at Execute.execute (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/commands/command.js:28:26)
       at Connection.handlePacket (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:449:32)
       at PacketParser.Connection.packetParser.p [as onPacket] (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:72:12)
       at PacketParser.executeStart (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/packet_parser.js:75:16)
       at Socket.Connection.stream.on.data (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:79:25)
       at Socket.emit (events.js:189:13)
       at addChunk (_stream_readable.js:284:12)
       at readableAddChunk (_stream_readable.js:265:11)
       at Socket.Readable.push (_stream_readable.js:220:10)
       at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
     code: 'ER_DUP_ENTRY',
     errno: 1062,
     sqlState: '23000',
     sqlMessage: 'Duplicate entry \'7\' for key \'roleId_UNIQUE\'',
     sql:
      'INSERT INTO `user` (`userId`,`roleId`,`username`,`adminname`,`password`,`createTime`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?,?,?,?);' },
  original:
   { Error: Duplicate entry '7' for key 'roleId_UNIQUE'
       at Packet.asError (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/packets/packet.js:684:17)
       at Execute.execute (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/commands/command.js:28:26)
       at Connection.handlePacket (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:449:32)
       at PacketParser.Connection.packetParser.p [as onPacket] (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:72:12)
       at PacketParser.executeStart (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/packet_parser.js:75:16)
       at Socket.Connection.stream.on.data (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:79:25)
       at Socket.emit (events.js:189:13)
       at addChunk (_stream_readable.js:284:12)
       at readableAddChunk (_stream_readable.js:265:11)
       at Socket.Readable.push (_stream_readable.js:220:10)
       at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
     code: 'ER_DUP_ENTRY',
     errno: 1062,
     sqlState: '23000',
     sqlMessage: 'Duplicate entry \'7\' for key \'roleId_UNIQUE\'',
     sql:
      'INSERT INTO `user` (`userId`,`roleId`,`username`,`adminname`,`password`,`createTime`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?,?,?,?);' },
  sql:
   'INSERT INTO `user` (`userId`,`roleId`,`username`,`adminname`,`password`,`createTime`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?,?,?,?);' }
[2019-07-10T11:49:11.646] [ERROR] user - Validation error { SequelizeUniqueConstraintError: Validation error
    at Query.formatError (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/dialects/mysql/query.js:218:16)
    at Execute.handler [as onResult] (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/dialects/mysql/query.js:46:23)
    at Execute.execute (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/commands/command.js:30:14)
    at Connection.handlePacket (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:449:32)
    at PacketParser.Connection.packetParser.p [as onPacket] (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:72:12)
    at PacketParser.executeStart (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/packet_parser.js:75:16)
    at Socket.Connection.stream.on.data (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:79:25)
    at Socket.emit (events.js:189:13)
    at addChunk (_stream_readable.js:284:12)
    at readableAddChunk (_stream_readable.js:265:11)
    at Socket.Readable.push (_stream_readable.js:220:10)
    at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
From previous event:
    at Query.run (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/dialects/mysql/query.js:39:12)
    at runHooks.then (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/sequelize.js:635:29)
From previous event:
    at Promise.try.then.connection (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/sequelize.js:635:12)
From previous event:
    at Promise.resolve.retry (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/sequelize.js:632:10)
    at /Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/retry-as-promised/index.js:70:21
    at new Promise (<anonymous>)
    at retryAsPromised (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/retry-as-promised/index.js:60:10)
    at Promise.try (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/sequelize.js:619:30)
From previous event:
    at Sequelize.query (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/sequelize.js:576:23)
    at QueryInterface.insert (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/query-interface.js:885:27)
    at Promise.try.then.then.then (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/model.js:3812:52)
    at runCallback (timers.js:705:18)
    at tryOnImmediate (timers.js:676:5)
    at processImmediate (timers.js:658:5)
From previous event:
    at model.save (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/model.js:3789:8)
    at Function.create (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/model.js:2216:8)
    at createUser (/Users/zhanghaiquan/Documents/my/amorphous_node/controller/user.js:102:23)
  name: 'SequelizeUniqueConstraintError',
  errors:
   [ ValidationErrorItem {
       message: 'roleId_UNIQUE must be unique',
       type: 'unique violation',
       path: 'roleId_UNIQUE',
       value: '7',
       origin: 'DB',
       instance: [user],
       validatorKey: 'not_unique',
       validatorName: null,
       validatorArgs: [] } ],
  fields: { roleId_UNIQUE: '7' },
  parent:
   { Error: Duplicate entry '7' for key 'roleId_UNIQUE'
       at Packet.asError (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/packets/packet.js:684:17)
       at Execute.execute (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/commands/command.js:28:26)
       at Connection.handlePacket (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:449:32)
       at PacketParser.Connection.packetParser.p [as onPacket] (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:72:12)
       at PacketParser.executeStart (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/packet_parser.js:75:16)
       at Socket.Connection.stream.on.data (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:79:25)
       at Socket.emit (events.js:189:13)
       at addChunk (_stream_readable.js:284:12)
       at readableAddChunk (_stream_readable.js:265:11)
       at Socket.Readable.push (_stream_readable.js:220:10)
       at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
     code: 'ER_DUP_ENTRY',
     errno: 1062,
     sqlState: '23000',
     sqlMessage: 'Duplicate entry \'7\' for key \'roleId_UNIQUE\'',
     sql:
      'INSERT INTO `user` (`userId`,`roleId`,`username`,`adminname`,`password`,`createTime`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?,?,?,?);' },
  original:
   { Error: Duplicate entry '7' for key 'roleId_UNIQUE'
       at Packet.asError (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/packets/packet.js:684:17)
       at Execute.execute (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/commands/command.js:28:26)
       at Connection.handlePacket (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:449:32)
       at PacketParser.Connection.packetParser.p [as onPacket] (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:72:12)
       at PacketParser.executeStart (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/packet_parser.js:75:16)
       at Socket.Connection.stream.on.data (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:79:25)
       at Socket.emit (events.js:189:13)
       at addChunk (_stream_readable.js:284:12)
       at readableAddChunk (_stream_readable.js:265:11)
       at Socket.Readable.push (_stream_readable.js:220:10)
       at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
     code: 'ER_DUP_ENTRY',
     errno: 1062,
     sqlState: '23000',
     sqlMessage: 'Duplicate entry \'7\' for key \'roleId_UNIQUE\'',
     sql:
      'INSERT INTO `user` (`userId`,`roleId`,`username`,`adminname`,`password`,`createTime`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?,?,?,?);' },
  sql:
   'INSERT INTO `user` (`userId`,`roleId`,`username`,`adminname`,`password`,`createTime`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?,?,?,?);' }
[2019-07-10T11:51:03.652] [ERROR] user - Validation error { SequelizeUniqueConstraintError: Validation error
    at Query.formatError (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/dialects/mysql/query.js:218:16)
    at Execute.handler [as onResult] (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/dialects/mysql/query.js:46:23)
    at Execute.execute (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/commands/command.js:30:14)
    at Connection.handlePacket (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:449:32)
    at PacketParser.Connection.packetParser.p [as onPacket] (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:72:12)
    at PacketParser.executeStart (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/packet_parser.js:75:16)
    at Socket.Connection.stream.on.data (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:79:25)
    at Socket.emit (events.js:189:13)
    at addChunk (_stream_readable.js:284:12)
    at readableAddChunk (_stream_readable.js:265:11)
    at Socket.Readable.push (_stream_readable.js:220:10)
    at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
From previous event:
    at Query.run (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/dialects/mysql/query.js:39:12)
    at runHooks.then (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/sequelize.js:635:29)
From previous event:
    at Promise.try.then.connection (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/sequelize.js:635:12)
From previous event:
    at Promise.resolve.retry (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/sequelize.js:632:10)
    at /Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/retry-as-promised/index.js:70:21
    at new Promise (<anonymous>)
    at retryAsPromised (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/retry-as-promised/index.js:60:10)
    at Promise.try (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/sequelize.js:619:30)
From previous event:
    at Sequelize.query (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/sequelize.js:576:23)
    at QueryInterface.insert (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/query-interface.js:885:27)
    at Promise.try.then.then.then (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/model.js:3812:52)
    at runCallback (timers.js:705:18)
    at tryOnImmediate (timers.js:676:5)
    at processImmediate (timers.js:658:5)
From previous event:
    at model.save (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/model.js:3789:8)
    at Function.create (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/model.js:2216:8)
    at createUser (/Users/zhanghaiquan/Documents/my/amorphous_node/controller/user.js:102:23)
  name: 'SequelizeUniqueConstraintError',
  errors:
   [ ValidationErrorItem {
       message: 'roleId_UNIQUE must be unique',
       type: 'unique violation',
       path: 'roleId_UNIQUE',
       value: '8',
       origin: 'DB',
       instance: [user],
       validatorKey: 'not_unique',
       validatorName: null,
       validatorArgs: [] } ],
  fields: { roleId_UNIQUE: '8' },
  parent:
   { Error: Duplicate entry '8' for key 'roleId_UNIQUE'
       at Packet.asError (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/packets/packet.js:684:17)
       at Execute.execute (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/commands/command.js:28:26)
       at Connection.handlePacket (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:449:32)
       at PacketParser.Connection.packetParser.p [as onPacket] (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:72:12)
       at PacketParser.executeStart (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/packet_parser.js:75:16)
       at Socket.Connection.stream.on.data (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:79:25)
       at Socket.emit (events.js:189:13)
       at addChunk (_stream_readable.js:284:12)
       at readableAddChunk (_stream_readable.js:265:11)
       at Socket.Readable.push (_stream_readable.js:220:10)
       at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
     code: 'ER_DUP_ENTRY',
     errno: 1062,
     sqlState: '23000',
     sqlMessage: 'Duplicate entry \'8\' for key \'roleId_UNIQUE\'',
     sql:
      'INSERT INTO `user` (`userId`,`roleId`,`username`,`adminname`,`password`,`createTime`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?,?,?,?);' },
  original:
   { Error: Duplicate entry '8' for key 'roleId_UNIQUE'
       at Packet.asError (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/packets/packet.js:684:17)
       at Execute.execute (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/commands/command.js:28:26)
       at Connection.handlePacket (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:449:32)
       at PacketParser.Connection.packetParser.p [as onPacket] (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:72:12)
       at PacketParser.executeStart (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/packet_parser.js:75:16)
       at Socket.Connection.stream.on.data (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:79:25)
       at Socket.emit (events.js:189:13)
       at addChunk (_stream_readable.js:284:12)
       at readableAddChunk (_stream_readable.js:265:11)
       at Socket.Readable.push (_stream_readable.js:220:10)
       at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
     code: 'ER_DUP_ENTRY',
     errno: 1062,
     sqlState: '23000',
     sqlMessage: 'Duplicate entry \'8\' for key \'roleId_UNIQUE\'',
     sql:
      'INSERT INTO `user` (`userId`,`roleId`,`username`,`adminname`,`password`,`createTime`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?,?,?,?);' },
  sql:
   'INSERT INTO `user` (`userId`,`roleId`,`username`,`adminname`,`password`,`createTime`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?,?,?,?);' }
[2019-07-10T11:55:21.826] [ERROR] user - Validation error { SequelizeUniqueConstraintError: Validation error
    at Query.formatError (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/dialects/mysql/query.js:218:16)
    at Execute.handler [as onResult] (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/dialects/mysql/query.js:46:23)
    at Execute.execute (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/commands/command.js:30:14)
    at Connection.handlePacket (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:449:32)
    at PacketParser.Connection.packetParser.p [as onPacket] (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:72:12)
    at PacketParser.executeStart (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/packet_parser.js:75:16)
    at Socket.Connection.stream.on.data (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:79:25)
    at Socket.emit (events.js:189:13)
    at addChunk (_stream_readable.js:284:12)
    at readableAddChunk (_stream_readable.js:265:11)
    at Socket.Readable.push (_stream_readable.js:220:10)
    at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
From previous event:
    at Query.run (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/dialects/mysql/query.js:39:12)
    at runHooks.then (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/sequelize.js:635:29)
From previous event:
    at Promise.try.then.connection (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/sequelize.js:635:12)
From previous event:
    at Promise.resolve.retry (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/sequelize.js:632:10)
    at /Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/retry-as-promised/index.js:70:21
    at new Promise (<anonymous>)
    at retryAsPromised (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/retry-as-promised/index.js:60:10)
    at Promise.try (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/sequelize.js:619:30)
From previous event:
    at Sequelize.query (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/sequelize.js:576:23)
    at QueryInterface.insert (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/query-interface.js:885:27)
    at Promise.try.then.then.then (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/model.js:3812:52)
    at runCallback (timers.js:705:18)
    at tryOnImmediate (timers.js:676:5)
    at processImmediate (timers.js:658:5)
From previous event:
    at model.save (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/model.js:3789:8)
    at Function.create (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/model.js:2216:8)
    at createUser (/Users/zhanghaiquan/Documents/my/amorphous_node/controller/user.js:102:23)
  name: 'SequelizeUniqueConstraintError',
  errors:
   [ ValidationErrorItem {
       message: 'roleId_UNIQUE must be unique',
       type: 'unique violation',
       path: 'roleId_UNIQUE',
       value: '7',
       origin: 'DB',
       instance: [user],
       validatorKey: 'not_unique',
       validatorName: null,
       validatorArgs: [] } ],
  fields: { roleId_UNIQUE: '7' },
  parent:
   { Error: Duplicate entry '7' for key 'roleId_UNIQUE'
       at Packet.asError (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/packets/packet.js:684:17)
       at Execute.execute (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/commands/command.js:28:26)
       at Connection.handlePacket (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:449:32)
       at PacketParser.Connection.packetParser.p [as onPacket] (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:72:12)
       at PacketParser.executeStart (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/packet_parser.js:75:16)
       at Socket.Connection.stream.on.data (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:79:25)
       at Socket.emit (events.js:189:13)
       at addChunk (_stream_readable.js:284:12)
       at readableAddChunk (_stream_readable.js:265:11)
       at Socket.Readable.push (_stream_readable.js:220:10)
       at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
     code: 'ER_DUP_ENTRY',
     errno: 1062,
     sqlState: '23000',
     sqlMessage: 'Duplicate entry \'7\' for key \'roleId_UNIQUE\'',
     sql:
      'INSERT INTO `user` (`userId`,`roleId`,`username`,`adminname`,`password`,`createTime`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?,?,?,?);' },
  original:
   { Error: Duplicate entry '7' for key 'roleId_UNIQUE'
       at Packet.asError (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/packets/packet.js:684:17)
       at Execute.execute (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/commands/command.js:28:26)
       at Connection.handlePacket (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:449:32)
       at PacketParser.Connection.packetParser.p [as onPacket] (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:72:12)
       at PacketParser.executeStart (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/packet_parser.js:75:16)
       at Socket.Connection.stream.on.data (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:79:25)
       at Socket.emit (events.js:189:13)
       at addChunk (_stream_readable.js:284:12)
       at readableAddChunk (_stream_readable.js:265:11)
       at Socket.Readable.push (_stream_readable.js:220:10)
       at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
     code: 'ER_DUP_ENTRY',
     errno: 1062,
     sqlState: '23000',
     sqlMessage: 'Duplicate entry \'7\' for key \'roleId_UNIQUE\'',
     sql:
      'INSERT INTO `user` (`userId`,`roleId`,`username`,`adminname`,`password`,`createTime`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?,?,?,?);' },
  sql:
   'INSERT INTO `user` (`userId`,`roleId`,`username`,`adminname`,`password`,`createTime`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?,?,?,?);' }
[2019-07-10T14:34:00.878] [ERROR] user - Validation error { SequelizeUniqueConstraintError: Validation error
    at Query.formatError (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/dialects/mysql/query.js:218:16)
    at Execute.handler [as onResult] (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/dialects/mysql/query.js:46:23)
    at Execute.execute (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/commands/command.js:30:14)
    at Connection.handlePacket (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:449:32)
    at PacketParser.Connection.packetParser.p [as onPacket] (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:72:12)
    at PacketParser.executeStart (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/packet_parser.js:75:16)
    at Socket.Connection.stream.on.data (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:79:25)
    at Socket.emit (events.js:189:13)
    at addChunk (_stream_readable.js:284:12)
    at readableAddChunk (_stream_readable.js:265:11)
    at Socket.Readable.push (_stream_readable.js:220:10)
    at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
From previous event:
    at Query.run (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/dialects/mysql/query.js:39:12)
    at runHooks.then (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/sequelize.js:635:29)
From previous event:
    at Promise.try.then.connection (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/sequelize.js:635:12)
From previous event:
    at Promise.resolve.retry (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/sequelize.js:632:10)
    at /Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/retry-as-promised/index.js:70:21
    at new Promise (<anonymous>)
    at retryAsPromised (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/retry-as-promised/index.js:60:10)
    at Promise.try (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/sequelize.js:619:30)
From previous event:
    at Sequelize.query (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/sequelize.js:576:23)
    at QueryInterface.insert (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/query-interface.js:885:27)
    at Promise.try.then.then.then (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/model.js:3812:52)
    at runCallback (timers.js:705:18)
    at tryOnImmediate (timers.js:676:5)
    at processImmediate (timers.js:658:5)
From previous event:
    at model.save (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/model.js:3789:8)
    at Function.create (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/model.js:2216:8)
    at createUser (/Users/zhanghaiquan/Documents/my/amorphous_node/controller/user.js:102:23)
  name: 'SequelizeUniqueConstraintError',
  errors:
   [ ValidationErrorItem {
       message: 'roleId_UNIQUE must be unique',
       type: 'unique violation',
       path: 'roleId_UNIQUE',
       value: '7',
       origin: 'DB',
       instance: [user],
       validatorKey: 'not_unique',
       validatorName: null,
       validatorArgs: [] } ],
  fields: { roleId_UNIQUE: '7' },
  parent:
   { Error: Duplicate entry '7' for key 'roleId_UNIQUE'
       at Packet.asError (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/packets/packet.js:684:17)
       at Execute.execute (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/commands/command.js:28:26)
       at Connection.handlePacket (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:449:32)
       at PacketParser.Connection.packetParser.p [as onPacket] (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:72:12)
       at PacketParser.executeStart (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/packet_parser.js:75:16)
       at Socket.Connection.stream.on.data (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:79:25)
       at Socket.emit (events.js:189:13)
       at addChunk (_stream_readable.js:284:12)
       at readableAddChunk (_stream_readable.js:265:11)
       at Socket.Readable.push (_stream_readable.js:220:10)
       at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
     code: 'ER_DUP_ENTRY',
     errno: 1062,
     sqlState: '23000',
     sqlMessage: 'Duplicate entry \'7\' for key \'roleId_UNIQUE\'',
     sql:
      'INSERT INTO `user` (`userId`,`roleId`,`username`,`adminname`,`password`,`createTime`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?,?,?,?);' },
  original:
   { Error: Duplicate entry '7' for key 'roleId_UNIQUE'
       at Packet.asError (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/packets/packet.js:684:17)
       at Execute.execute (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/commands/command.js:28:26)
       at Connection.handlePacket (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:449:32)
       at PacketParser.Connection.packetParser.p [as onPacket] (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:72:12)
       at PacketParser.executeStart (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/packet_parser.js:75:16)
       at Socket.Connection.stream.on.data (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:79:25)
       at Socket.emit (events.js:189:13)
       at addChunk (_stream_readable.js:284:12)
       at readableAddChunk (_stream_readable.js:265:11)
       at Socket.Readable.push (_stream_readable.js:220:10)
       at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
     code: 'ER_DUP_ENTRY',
     errno: 1062,
     sqlState: '23000',
     sqlMessage: 'Duplicate entry \'7\' for key \'roleId_UNIQUE\'',
     sql:
      'INSERT INTO `user` (`userId`,`roleId`,`username`,`adminname`,`password`,`createTime`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?,?,?,?);' },
  sql:
   'INSERT INTO `user` (`userId`,`roleId`,`username`,`adminname`,`password`,`createTime`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?,?,?,?);' }
[2019-07-10T14:43:29.209] [ERROR] user - Validation error { SequelizeUniqueConstraintError: Validation error
    at Query.formatError (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/dialects/mysql/query.js:218:16)
    at Execute.handler [as onResult] (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/dialects/mysql/query.js:46:23)
    at Execute.execute (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/commands/command.js:30:14)
    at Connection.handlePacket (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:449:32)
    at PacketParser.Connection.packetParser.p [as onPacket] (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:72:12)
    at PacketParser.executeStart (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/packet_parser.js:75:16)
    at Socket.Connection.stream.on.data (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:79:25)
    at Socket.emit (events.js:189:13)
    at addChunk (_stream_readable.js:284:12)
    at readableAddChunk (_stream_readable.js:265:11)
    at Socket.Readable.push (_stream_readable.js:220:10)
    at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
From previous event:
    at Query.run (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/dialects/mysql/query.js:39:12)
    at runHooks.then (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/sequelize.js:635:29)
From previous event:
    at Promise.try.then.connection (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/sequelize.js:635:12)
From previous event:
    at Promise.resolve.retry (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/sequelize.js:632:10)
    at /Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/retry-as-promised/index.js:70:21
    at new Promise (<anonymous>)
    at retryAsPromised (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/retry-as-promised/index.js:60:10)
    at Promise.try (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/sequelize.js:619:30)
From previous event:
    at Sequelize.query (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/sequelize.js:576:23)
    at QueryInterface.insert (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/query-interface.js:885:27)
    at Promise.try.then.then.then (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/model.js:3812:52)
    at runCallback (timers.js:705:18)
    at tryOnImmediate (timers.js:676:5)
    at processImmediate (timers.js:658:5)
From previous event:
    at model.save (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/model.js:3789:8)
    at Function.create (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/model.js:2216:8)
    at createUser (/Users/zhanghaiquan/Documents/my/amorphous_node/controller/user.js:102:23)
  name: 'SequelizeUniqueConstraintError',
  errors:
   [ ValidationErrorItem {
       message: 'roleId_UNIQUE must be unique',
       type: 'unique violation',
       path: 'roleId_UNIQUE',
       value: '7',
       origin: 'DB',
       instance: [user],
       validatorKey: 'not_unique',
       validatorName: null,
       validatorArgs: [] } ],
  fields: { roleId_UNIQUE: '7' },
  parent:
   { Error: Duplicate entry '7' for key 'roleId_UNIQUE'
       at Packet.asError (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/packets/packet.js:684:17)
       at Execute.execute (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/commands/command.js:28:26)
       at Connection.handlePacket (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:449:32)
       at PacketParser.Connection.packetParser.p [as onPacket] (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:72:12)
       at PacketParser.executeStart (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/packet_parser.js:75:16)
       at Socket.Connection.stream.on.data (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:79:25)
       at Socket.emit (events.js:189:13)
       at addChunk (_stream_readable.js:284:12)
       at readableAddChunk (_stream_readable.js:265:11)
       at Socket.Readable.push (_stream_readable.js:220:10)
       at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
     code: 'ER_DUP_ENTRY',
     errno: 1062,
     sqlState: '23000',
     sqlMessage: 'Duplicate entry \'7\' for key \'roleId_UNIQUE\'',
     sql:
      'INSERT INTO `user` (`userId`,`roleId`,`username`,`adminname`,`password`,`createTime`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?,?,?,?);' },
  original:
   { Error: Duplicate entry '7' for key 'roleId_UNIQUE'
       at Packet.asError (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/packets/packet.js:684:17)
       at Execute.execute (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/commands/command.js:28:26)
       at Connection.handlePacket (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:449:32)
       at PacketParser.Connection.packetParser.p [as onPacket] (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:72:12)
       at PacketParser.executeStart (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/packet_parser.js:75:16)
       at Socket.Connection.stream.on.data (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:79:25)
       at Socket.emit (events.js:189:13)
       at addChunk (_stream_readable.js:284:12)
       at readableAddChunk (_stream_readable.js:265:11)
       at Socket.Readable.push (_stream_readable.js:220:10)
       at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
     code: 'ER_DUP_ENTRY',
     errno: 1062,
     sqlState: '23000',
     sqlMessage: 'Duplicate entry \'7\' for key \'roleId_UNIQUE\'',
     sql:
      'INSERT INTO `user` (`userId`,`roleId`,`username`,`adminname`,`password`,`createTime`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?,?,?,?);' },
  sql:
   'INSERT INTO `user` (`userId`,`roleId`,`username`,`adminname`,`password`,`createTime`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?,?,?,?);' }
[2019-07-10T14:44:32.467] [ERROR] user - Validation error { SequelizeUniqueConstraintError: Validation error
    at Query.formatError (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/dialects/mysql/query.js:218:16)
    at Execute.handler [as onResult] (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/dialects/mysql/query.js:46:23)
    at Execute.execute (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/commands/command.js:30:14)
    at Connection.handlePacket (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:449:32)
    at PacketParser.Connection.packetParser.p [as onPacket] (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:72:12)
    at PacketParser.executeStart (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/packet_parser.js:75:16)
    at Socket.Connection.stream.on.data (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:79:25)
    at Socket.emit (events.js:189:13)
    at addChunk (_stream_readable.js:284:12)
    at readableAddChunk (_stream_readable.js:265:11)
    at Socket.Readable.push (_stream_readable.js:220:10)
    at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
From previous event:
    at Query.run (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/dialects/mysql/query.js:39:12)
    at runHooks.then (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/sequelize.js:635:29)
From previous event:
    at Promise.try.then.connection (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/sequelize.js:635:12)
From previous event:
    at Promise.resolve.retry (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/sequelize.js:632:10)
    at /Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/retry-as-promised/index.js:70:21
    at new Promise (<anonymous>)
    at retryAsPromised (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/retry-as-promised/index.js:60:10)
    at Promise.try (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/sequelize.js:619:30)
From previous event:
    at Sequelize.query (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/sequelize.js:576:23)
    at QueryInterface.insert (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/query-interface.js:885:27)
    at Promise.try.then.then.then (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/model.js:3812:52)
    at runCallback (timers.js:705:18)
    at tryOnImmediate (timers.js:676:5)
    at processImmediate (timers.js:658:5)
From previous event:
    at model.save (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/model.js:3789:8)
    at Function.create (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/sequelize/lib/model.js:2216:8)
    at createUser (/Users/zhanghaiquan/Documents/my/amorphous_node/controller/user.js:102:23)
  name: 'SequelizeUniqueConstraintError',
  errors:
   [ ValidationErrorItem {
       message: 'roleId_UNIQUE must be unique',
       type: 'unique violation',
       path: 'roleId_UNIQUE',
       value: '7',
       origin: 'DB',
       instance: [user],
       validatorKey: 'not_unique',
       validatorName: null,
       validatorArgs: [] } ],
  fields: { roleId_UNIQUE: '7' },
  parent:
   { Error: Duplicate entry '7' for key 'roleId_UNIQUE'
       at Packet.asError (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/packets/packet.js:684:17)
       at Execute.execute (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/commands/command.js:28:26)
       at Connection.handlePacket (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:449:32)
       at PacketParser.Connection.packetParser.p [as onPacket] (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:72:12)
       at PacketParser.executeStart (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/packet_parser.js:75:16)
       at Socket.Connection.stream.on.data (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:79:25)
       at Socket.emit (events.js:189:13)
       at addChunk (_stream_readable.js:284:12)
       at readableAddChunk (_stream_readable.js:265:11)
       at Socket.Readable.push (_stream_readable.js:220:10)
       at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
     code: 'ER_DUP_ENTRY',
     errno: 1062,
     sqlState: '23000',
     sqlMessage: 'Duplicate entry \'7\' for key \'roleId_UNIQUE\'',
     sql:
      'INSERT INTO `user` (`userId`,`roleId`,`username`,`adminname`,`password`,`createTime`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?,?,?,?);' },
  original:
   { Error: Duplicate entry '7' for key 'roleId_UNIQUE'
       at Packet.asError (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/packets/packet.js:684:17)
       at Execute.execute (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/commands/command.js:28:26)
       at Connection.handlePacket (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:449:32)
       at PacketParser.Connection.packetParser.p [as onPacket] (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:72:12)
       at PacketParser.executeStart (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/packet_parser.js:75:16)
       at Socket.Connection.stream.on.data (/Users/zhanghaiquan/Documents/my/amorphous_node/node_modules/mysql2/lib/connection.js:79:25)
       at Socket.emit (events.js:189:13)
       at addChunk (_stream_readable.js:284:12)
       at readableAddChunk (_stream_readable.js:265:11)
       at Socket.Readable.push (_stream_readable.js:220:10)
       at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
     code: 'ER_DUP_ENTRY',
     errno: 1062,
     sqlState: '23000',
     sqlMessage: 'Duplicate entry \'7\' for key \'roleId_UNIQUE\'',
     sql:
      'INSERT INTO `user` (`userId`,`roleId`,`username`,`adminname`,`password`,`createTime`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?,?,?,?);' },
  sql:
   'INSERT INTO `user` (`userId`,`roleId`,`username`,`adminname`,`password`,`createTime`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?,?,?,?);' }
[2019-07-10T15:46:52.096] [ERROR] appearenceLevel - 带材外观值重复 Error: 带材外观值重复
    at createData (/Users/zhanghaiquan/Documents/my/amorphous_node/controller/appearenceLevel.js:57:15)
